src/app/shared/directives/index.ts

```diff
@@ -0,0 +1,3 @@
+export { IsAuthorizedDirective } from "./is-authorized/is-authorized.directive";
+export { BorderDecoratorDirective } from "./border-decorator/border-decorator.directive";
+export { DirectivesModule } from "./DirectivesModule";

```

Comment:
files should be called in kebab case
directives.module.ts

Reply:
fixed